server/src/assets/game/tableGenerator/Readme.md

```diff
@@ -0,0 +1,23 @@
+# Table Generator for SWTbahn Game
+
+This application generates the configuration json files for the swtbahn game client, which are required for the signal-symbol mapping and the route suggestion mechanics.
+Once entered the signals to symbol, the programm will automatically find a suitable route (which the least nodes) and store it for the game client.
```

Comment:
"Once entered the signals to symbol" - I don't understand what that really means.